iphone-and-ipad/index.php

```diff
@@ -82,6 +82,12 @@
   </tbody>
 </table>
 
+<h2>New in Keyman 15.0 (Apr 2022)</h2>
+<ul>
+  <li>General polish, bug fixes, and performance improvements</li>
+  <li>Update minimum iOS version to 12.1</li>
```

Comment:
Same as earlier on 'General polish'. Bug fixes and peformance improvements are clear, but 'polish' is a little vague. I'm not really sure what it means.

Reply:
See my suggestion above